### PR TITLE
Feat: Advanced AI Analytics

### PR DESCRIPTION
**Pull request by Guaris**
https://konghq.atlassian.net/browse/DOCU-4055
https://konghq.atlassian.net/browse/DOCU-4050

### CODE REVIEW

File: app/konnect/analytics/explorer.md [hunk 1]
```diff
@@ -40,6 +46,101 @@ Kong latency | Latency | The amount of time, in milliseconds, that {{site.base_g
...
+{% navtab Grouping and Filtering %}
+
+This is an overview of the different grouping and filtering categories available for either **API usage** or **LLM usage** : 
+
+* **Provider**                                                         
+* **Request Model**                                                   
+* **Response Model**                                          
+* **Cache Status**                                                         
+* **Embeddings Provider**                                                  
+* **Embeddings Name**
```
Comment (sichvoge) [inline app/konnect/analytics/explorer.md:108]:
```suggestion
* **Embeddings Model**                                                   
```

**Review by sichvoge — CHANGES_REQUESTED** [review]

File: app/konnect/analytics/explorer.md [hunk 2]
```diff
@@ -40,6 +44,100 @@ Kong latency | Latency | The amount of time, in milliseconds, that {{site.base_g
...
+one-second resolution data, while longer time frames display minute, hour, or days resolution data.
+
+All time interval presets are **relative**. 
+For custom reports, you can also choose a **custom** date range.
+
+* **Relative** time frames are dynamic and the report captures a snapshot of data
+relative to when a user views the report.
+* **Custom** time frames are static and the report captures a snapshot of data
+during the specified time frame. You can see the exact range below
+the time frame selector. For example:
```
Comment (cloudjumpercat) [inline app/konnect/analytics/explorer.md:61]:
```suggestion
All time interval presets are **relative**. This means that time frames are dynamic and the report captures a snapshot of data
relative to when a user views the report.
For custom reports, you can also choose a **custom** date range. Custom means that time frames are static and the report captures a snapshot of data
during the specified time frame. You can see the exact range below
the time frame selector. For example:
```

File: app/konnect/analytics/explorer.md [hunk 3]
```diff
@@ -40,6 +44,100 @@ Kong latency | Latency | The amount of time, in milliseconds, that {{site.base_g
...
+Last 30 days | Data is aggregated in daily increments.
+Current week | Data is aggregated in one hour increments. Logs any traffic in the current calendar week. 
+Current month | Data is aggregated in one hour increments. Logs any traffic in the current calendar month. 
+Previous week | Data is aggregated in one hour increments. Logs any traffic in the previous calendar week.
+Previous month | Data is aggregated in daily increments. Logs any traffic in the previous calendar month. 
+
+{% endnavtab %}
+{% endnavtabs %}
+
+## LLM usage reporting
```
Comment (cloudjumpercat) [inline app/konnect/analytics/explorer.md:84]: Where is the LLM info coming from? Does a user need to configure AI Gateway plugins before this data will start streaming into Konnect?

File: app/konnect/analytics/index.md [hunk 4]
```diff
@@ -29,10 +29,13 @@ You can use the following table to help you determine which analytics tools are
...
 | -------------- | ----------- |
 | Quickly understand API usage and performance at a glance, and gain insights across your entire organization. | [Summary dashboard](/konnect/analytics/dashboard/) |
-| Browse API usage data to gain access to key performance and health statistics. Visualize, slice, and dice API usage data you’ve stored in only a few clicks. | [Explorer](/konnect/analytics/explorer/) |
+| Browse API usage data to gain access to key performance and health statistics. Visualize, slice, and dice API usage data you’ve stored in only a few clicks. | [Explorer](/konnect/analytics/explorer/#api-usage-reporting) |
 | Communicate insights across teams or departments and share important KPIs. | [Custom reports](/konnect/analytics/custom-reports/) |
 | Understand user behavior or pin point where issues occur. See detailed information about each individual request made to your APIs. | [Requests](/konnect/analytics/api-requests/)  |
 | Manage Analytics cost for individual control planes| [Data ingestion](/konnect/analytics/#data-ingestion) |
+| Understand LLM usage, tokens and costs | [Explorer](/konnect/analytics/explorer/#llm-usage-reporting) |
```
Comment (cloudjumpercat) [inline app/konnect/analytics/index.md:36]:
```suggestion
| Understand LLM usage, tokens, and costs | [Explorer](/konnect/analytics/explorer/#llm-usage-reporting) | 
```

**Review by cloudjumpercat — APPROVED** [review]
Left a few small nitpick comments!

File: app/konnect/analytics/explorer.md (hunk 3 above)
Comment (sichvoge) [reply app/konnect/analytics/explorer.md:81]: Yes.

File: app/konnect/analytics/explorer.md [hunk 5]
```diff
@@ -40,6 +44,100 @@ Kong latency | Latency | The amount of time, in milliseconds, that {{site.base_g
...
+With LLM usage reporting, you can:
+
+* Track token consumption: Monitor the number of tokens processed by the different LLM models you have configured. 
+* Understand costs: Gain visibility into the costs associated with your LLM providers. 
+* Measure Latency: Analyze the latency involved in processing LLM requests. 
+
+To use this feature, navigate to the [Explorer dashboard](https://cloud.konghq.com/us/analytics/explorer) and switch between API usage and LLM usage using the new dataset dropdown. Metrics and groupings will dynamically adjust based on the selected dataset. 
+
+{:.note}
+>**Note**: LLM usage reporting requires {{site.base_gateway}} version 3.8.
```
Comment (Guaris) [inline app/konnect/analytics/explorer.md:94]:
```suggestion
>**Note**: LLM usage reporting requires at least {{site.base_gateway}} version 3.8. 
```

**Review by sichvoge — APPROVED** [review]

**Pull request merged**